src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php

```diff
@@ -126,7 +126,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $builder->setTimeout(null);
         $process = $builder->getProcess();
 
-        if (OutputInterface::VERBOSITY_VERBOSE > $output->getVerbosity()) {
```

Comment:
You are probably right. Using the original object will give better performance than the wrapped one.
DONE